compiler/src/iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp

```diff
@@ -186,6 +210,19 @@ struct LinalgFusePass : public LinalgFuseBase<LinalgFusePass> {
   void runOnOperation() override;
 };
 
+struct LinalgSplitReductionPass
+    : public LinalgSplitReductionBase<LinalgSplitReductionPass> {
+  LinalgSplitReductionPass(bool enableReductionReordering, int64_t size = 0,
+                           uint64_t unrollFactor = 1) {
```

Comment:
unrollFactor is not used. It seems unrelated to the PR. How about removing it and reintroducing when there's a need?